### DISCUSSION TITLE
Wifi feature gone.

### DISCUSSION THREAD

**Discussion by PeteyPiper** [Guilouz/Klipper-Flsun-Speeder-Pad#171] [Q&A]
Had my modem reset. Then go try to printer via. Browser. It’s not connecting. Check pad for ip address change. But instead of wireless menu. I’m displaying this on pad. 
![IMG_1133](https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/assets/139482842/cb9a6337-bd43-4016-84d5-06afb01eee32)
![IMG_1134](https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/assets/139482842/9a37fde1-bf02-43d1-9654-21403f560b54)
Please someone assist on how to fix.

**Comment by najwaa-debug** [comment]
Same issue. I can still connect to the printer via the browser but the network section on the pad shows this message.

**Comment by Guilouz** [comment]
There is some changes in latest klipperscreen version.
It’s needed to restore pad and follow the new guide.
